guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-manually.adoc

```diff
@@ -0,0 +1,63 @@
+[id="deploying-the-ca-certificate-on-a-host-manually"]
+= Deploying the CA certificate on a host manually
+
+You can deploy the CA certificate on the host manually by rendering a public provisioning template, which will provide the certificate to your host.
+
+.Prerequisites
+* You have root access on your host.
+* The CA certificate has been changed on {ProjectServer}.
+For more information, see {AdministeringDocURL}planning-for-ca-certificate-renewal[Planning for CA certificate renewal] in _{AdministeringDocTitle}_.
+
+.Procedure
+. Log in to you host by using SSH.
+. Download the certificate on your host:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# curl -o "{project-context}_ca_cert.crt" https://_{foreman-example-com}_/unattended/public/foreman_raw_ca
+----
++
+Replace _{foreman-example-com}_ with the hostname of your {ProjectServer}.
++
+You can use HTTP when the CA certificate is expired.
+However, {Team} recommends preparing the certificate renewal in advance.
+. Copy the certificate to the truststore:
+ifdef::client-content-dnf[]
+ifndef::satellite[]
+* On {EL}:
+endif::[]
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# cp {project-context}_ca_cert.crt /etc/pki/ca-trust/source/anchors
```

Comment:
This is not necessary.

Reply:
And is only for some uses cases, like container content access.

Reply:
@ehelms  Are you sure? If we update the certs manually we need to copy the cert to the truststore, how it will get to the truststore otherwise?

Reply:
And if you are talking generally, I suppose having the manual procedure outlined is a good thing.

Reply:
My point is, the cert only needs to go in the truststore if it needs to go in there. We shouldn't be blindly putting it in the truststore in *all* cases, just the ones where it's required.

Reply:
@ehelms When does the CA cert need to go to the truststore?

Reply:
When the server CA is not already in the trust store. Let's say your server uses Let's Encrypt (or any other well known CA) for certificates, then you can expect it to already be in the trust store and adding it really is redundant.

Reply:
@ianballou Is this still true for docker? for podman? Do we have to add the CA to the system wide trust store? Or can those services be configured individually to use the CA certificate?

Reply:
@ekohl if Foreman uses a well-known CA, then the whole process is redundant. 

@Lennonka What do you think, should we add a "precodition" that this whole process (the process of updating hosts with the new CA) is relevant if the certificate that is used by Foreman is not trusted by default?